Pods/CatchingFire/src/CatchingFire.swift

```diff
@@ -8,6 +8,10 @@
 
 import XCTest
 
+#if !swift(>=3.0)
```

Comment:
Instead of using a negative, what about about using a different comparison? I think this is more readable and clear.

```swift
#if swift(<3.0)
```